saleor/graphql/checkout/tests/mutations/test_checkout_shipping_method_update_nullable_shipping_method_id.py

```diff
@@ -0,0 +1,67 @@
+from unittest.mock import patch
+
+import pytest
+
+from ....core.utils import to_global_id_or_none
+from ....tests.utils import get_graphql_content
+
+MUTATION_UPDATE_SHIPPING_METHOD = """
+    mutation checkoutShippingMethodUpdate(
+        $id: ID, $shippingMethodId: ID
+    ) {
+        checkoutShippingMethodUpdate(
+            id: $id, shippingMethodId: $shippingMethodId
+        ) {
+            errors {
+                field
+                message
+                code
+            }
+            checkout {
+                id
+                token
+                shippingMethod {
+                    id
+                }
+            }
+        }
+    }
+"""
+
+
+@pytest.mark.django_db
+@pytest.mark.count_queries(autouse=False)
+@patch(
+    "saleor.graphql.checkout.mutations.checkout_shipping_method_update."
+    "clean_delivery_method"
+)
+def test_checkout_shipping_method_update_nullable_shipping_method_id(
+    mock_clean_delivery_method,
+    staff_api_client,
+    checkout_with_item_and_shipping_method,
+    count_queries,
```

Comment:
Those lines are required for benchmarks. We should use them outside the `benchmark` module. To avoid mixing unit tests with benchmarks. 
```suggestion
@pytest.mark.django_db
@patch(
    "saleor.graphql.checkout.mutations.checkout_shipping_method_update."
    "clean_delivery_method"
)
def test_checkout_shipping_method_update_nullable_shipping_method_id(
    mock_clean_delivery_method,
    staff_api_client,
    checkout_with_item_and_shipping_method,
```